### ISSUE TITLE
Color contrast

### ISSUE THREAD

**Issue by zaladeokin** [Piwigo/Piwigo#1810]
Color contrast of the inactive tab should be improved for good accessibility.

**Comment by plegall** [comment]
Hi @zaladeokin 

we've briefly discussed your issue, and Alice (our designer) says she globally agrees about the lack of contrast, but she would like to get some details about where you would like more contrast (screenshot welcomed)

**Comment by zaladeokin** [comment]
![piwigo](https://user-images.githubusercontent.com/110726099/205183722-d8800102-2f23-4634-8670-bd1cd4aea44b.png)
The color contrast of the elements circle in the picture attached should be adjusted.

**Comment by plegall** [comment]
The contrast problem is on the background color of inactive tabs or text color of inactive tabs?

**Comment by Phil353556** [comment]
My 2cts: answer is both

**Comment by plegall** [comment]
Info : the inactive tabs have no background color. The contrast is zero compared to tabs background. That's on purpose.

Contrast ratio of inactive tabs text `#898989` compared to background `#f3f3f3` is 3.15. It's not enough indeed. Recommendation is 4.5. It would require to darken text to `#6f6f6f` (ratio=4.52).

We have started an in-depth analysis of contrast-ratio issues in the administrtion.
